pysatNASA/instruments/icon_mighti.py

```diff
@@ -41,24 +41,31 @@
                               keep_original_names=True)
 
 Authors
----------
+-------
 Originated from EUV support.
 Jeff Klenzing, Mar 17, 2018, Goddard Space Flight Center
 Russell Stoneback, Mar 23, 2018, University of Texas at Dallas
-Conversion to MIGHTI, Oct 8th, 2028, University of Texas at Dallas
+Conversion to MIGHTI, Oct 8th, 2018, University of Texas at Dallas
+
+Note
+----
+Currently red and green data products are bundled together in zip files on the
+server.  This results in 'double downloading'.  This will be fixed once data is
+transfered to SPDF.
 
 """
 
 import datetime as dt
 import functools
-import logging
 
 import pysat
+from pysat import logger
 from pysat.instruments.methods import general as mm_gen
+from pysatNASA.instruments.methods import cdaweb as cdw
```

Comment:
```suggestion

from pysatNASA.instruments.methods import cdaweb as cdw
```

Reply:
This was a 'here and elsewhere' that wasn't carried through.  Recommend checking other files.

Reply:
updated throughout